src/dtx/dtx_rpc.c

```diff
@@ -80,8 +81,12 @@ struct dtx_req_rec {
 	int				 drr_count; /* DTX count */
 	int				 drr_result; /* The RPC result */
 	uint32_t			 drr_comp:1;
+	uint32_t			 drr_bitmap_sz;
 	struct dtx_id			*drr_dti; /* The DTX array */
-	uint32_t			*drr_flags;
+	union {
+		uint32_t		*drr_flags;
+		uint8_t			*drr_bitmap;
```

Comment:
just confirm that it is impossible to use both drr_flags and bitmap in future?

Reply:
drr_flags is only for DTX refresh RPC. drr_bitmap is only used for collectively commit/abort DTX. They are different DTX RPCs, will not be valid for the same time.